<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myTime</class>
 <widget class="QDialog" name="myTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myTime</string>
  </property>
  <widget class="QLabel" name="labTimer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>testList</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>432</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>432</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
